stock-tracker/src/components/Header.js

```diff
@@ -58,9 +86,11 @@ const Header = () => {
           }
           type="text"
           value={symbol}
-          onChange={e => setSymbol(e.target.value)}
+          onChange={onChange}
           onKeyPress={handleSubmit}
+          // onKeyDown={onKeyDown}
         />
+        <ul tabIndex="0" className='suggestion_list' style={{display: open ? 'block' : 'none'}}>{getSuggestions}</ul>
```

Comment:
Tried this and when I clicked away from the search to close the dropdown, I got a type error that dropdownRef.current is null since clicking away causes open to be set to false therefore not rendering the <ul> with a ref